src/components/commons/RoundedTable.tsx

```
@@ -0,0 +1,353 @@
+  assignments: {
+    headers: ['번호', '구분', '과제명', '제출시기'],
+    columns: ['index', 'type', 'name', 'dueDate']
+  },
+  evaluation: {
+    headers: ['항목', '비중', '만점', '공개여부', '설명'],
+    columns: ['item', 'weight', 'maxScore', 'isPublic', 'description']
+  }
+};
+
+// 기본값 생성 함수
+export const createDefaultData = (type: string, _count: number = 1) => {
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)